python/utils.py

```diff
@@ -28,14 +28,19 @@ def load_all_yaml_files() -> Dict[str, Dict[str, Dict]]:
     for transform_type in constants.TRANSFORM_TYPES:
         transform_type_dir_path = _get_udt_repo_dir() / f"{transform_type}_transforms"
 
-        # Get list of all transform of certain type
-        transform_names = [x.name for x in transform_type_dir_path.rglob("*/**")]
+        # Get list of all transform names of this type,
+        # by looking at sub-directory names one level down
+        transform_names = [
+            x.name for x in transform_type_dir_path.iterdir() if x.is_dir()
+        ]
```

Comment:
@griffatrasgo FYI, we may want to change the `RasgoQL` packages to get transform names of a certain type this way as well.

Doing `transform_type_dir_path.rglob("*/**")` would return all subdirectories recursively, so we were trying to load YAML files that didn't exist causing the exception to happen.